lib/migration.pm

```diff
@@ -207,4 +209,19 @@ sub set_scc_proxy_url {
     save_screenshot;
 }
 
+# Enable ZYPP_SINGLE_RPMTRANS=1 if the OS have
+# libzypp-17.28.4 and zypper-1.14.49 or higher version installed
+sub set_zypp_single_rpmtrans {
+    my $zypper_ver = script_output q(rpm -ivq zypper | awk -F\- '{print $2}');
+    my $libzypp_ver = script_output q(rpm -ivq libzypp | awk -F\- '{print $2}');
+
+    if ((package_version_cmp($zypper_ver, '1.14.49') >= 0) &&
+        (package_version_cmp($libzypp_ver, '17.28.4') >= 0)) {
+        assert_script_run 'export ZYPP_SINGLE_RPMTRANS=1';
+        assert_script_run 'echo techpreview.ZYPP_SINGLE_RPMTRANS=1 >> /etc/zypp/zypp.conf';
```

Comment:
I suppose we only need select one line of 220 and 221?

Reply:
changed.